submissions/Roka20012/js-dom/interactive.js

```diff
@@ -2,52 +2,49 @@
-				<p><span>Eligendi architecto voluptas alias labore reprehenderit temporibus sed, et laborum repellendus quis quae dicta mollitia voluptates, hic? Excepturi praesentium vel, labore veniam earum facere quae deserunt eius quaerat deleniti quod.</span>
-				<span>Qui asperiores assumenda, magni hic magnam quis earum blanditiis dolor adipisci, quibusdam voluptatibus ducimus distinctio quidem eaque dolorum voluptate delectus fuga nisi beatae veritatis. Debitis optio aspernatur deserunt commodi! Saepe.</span></p>`
+                <span>Hic nesciunt nemo quia odio quo numquam cum praesentium quis dolor impedit amet porro aut eos eaque dolore esse incidunt quos, voluptas doloremque eveniet blanditiis! Earum doloremque temporibus cupiditate laboriosam.</span>
+                <span>Doloribus fugiat cumque nostrum, error iusto vitae voluptatem porro ipsam officiis provident ut ad. Repellat quas at temporibus numquam rem culpa adipisci, blanditiis soluta vitae accusantium, veritatis laboriosam. Voluptate, praesentium.</span>
+                <span>Laborum repudiandae id, quasi voluptate animi nulla, molestias provident possimus, architecto accusamus magnam veniam rerum perspiciatis dicta assumenda unde et eveniet sed illum quaerat aut eum suscipit! Deserunt voluptas, doloribus.</span>
+                <span>Rem ipsum eligendi id consequatur, expedita cumque neque. Dolor natus tempora repellendus quaerat accusamus eligendi doloribus, accusantium dolore voluptatibus dolorem fugiat architecto dolorum inventore error! Magnam doloremque velit, commodi minima!</span></p>
+                <p><span>Eligendi architecto voluptas alias labore reprehenderit temporibus sed, et laborum repellendus quis quae dicta mollitia voluptates, hic? Excepturi praesentium vel, labore veniam earum facere quae deserunt eius quaerat deleniti quod.</span>
+                <span>Qui asperiores assumenda, magni hic magnam quis earum blanditiis dolor adipisci, quibusdam voluptatibus ducimus distinctio quidem eaque dolorum voluptate delectus fuga nisi beatae veritatis. Debitis optio aspernatur deserunt commodi! Saepe.</span></p>`
 };
 
 
 let nav = document.getElementById("navbar");
 let text = document.getElementById("text");
 let burger = document.getElementsByClassName("burger2")[0];
-
 /**
  * закриває меню якщо ми натискаємо на пункт меню 
  * коли ширина екрана менша 900px
  * author: Rostyk Broslavsky
  * email: broslavsky.rostislaw@gmail.com
  */
-function hideBurgerMenu() {
-    let width = window.innerWidth ||
-        document.documentElement.clientWidth ||
-        document.body.clientWidth;
-    let maxWidth = 900;
-    if (width < maxWidth) {
+function hideBurgerMenu(target) {
+    let computedStyle = getComputedStyle(burger);
+
+    if (computedStyle.left == "15px") {
         nav.classList.toggle("menu-open");
         nav.classList.toggle("menu-close");
```

Comment:
Maybe you can use one class here(only 'open' for example)?

Reply:
Did it)